docs/ChangeLog/20250824.md

```diff
@@ -0,0 +1,23 @@
+# QMK Breaking Changes - 2025 August 24 Changelog
+
+## Notable Features
+
+### Tap Dance ([#25415](https://github.com/qmk/qmk_firmware/pull/25415))
+
+Tap dance state has been removed from `tap_dance_action_t`. Instead, tap dance
+state can now be obtained from a method `tap_dance_get_state(int tap_dance_idx)`.
+
+`action.state->xxxxx` should generally be replaced with:
+
+```c
+tap_dance_state_t* state;
+...
+state = tap_dance_get_state(QK_TAP_DANCE_GET_INDEX(keycode));
+...
+state->xxxxx
+```
+   
+Eg. [Example 3 in the tap dance documentation](../tap_dance#example-3).
```

Comment:
```suggestion
Eg. [Example 3 in the tap dance documentation](/features/tap_dance#example-3).
```

Reply:
Also this file should just be a stub under `<qmk_firmware>/docs/Changelog/20250831/PR25415.md`, rather than starting the changelog file itself.

Reply:
Another couple of notes,
* The change is not something for the "Notable Features` section and would need to be within a `Changes Requiring User Action` section.
* The lines should not be manually wrapped